ant-design/src/theme-overrides.less

```diff
@@ -0,0 +1,384 @@
+html,
+body {
+  font-weight: 300;
+  letter-spacing: 0.01em;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: 200;
+  letter-spacing: 0;
+}
+
+h1, h2 { font-size: 33px; }
+
+// -------- Colors -----------
+
+@primary-color: @blue-60;
+
+@white: #FFF; //@grey-10;
+@black: @ink-90;
+
+@blue-6: @blue-50;
+@purple-6: @purple-50;
+@cyan-6: @teal-50;
+@green-6: @green-60;
+@pink-6: @magenta-50;
+@red-6: @red-60;
+@orange-6: @orange-60;
+@yellow-6: @yellow-60;
+
+// Ant colors
+@info-color             : @blue-80;
+@success-color          : @green-60;
+@error-color            : @red-50;
+@highlight-color        : @orange-50;
+@warning-color          : @yellow-50;
+@normal-color           : #d9d9d9;
+
+// Base Scaffolding Variables
+// ---
+
+// Background color for `<body>`
+@body-background        : @white;
+// Base background color for most components
+@component-background   : @white;
+@font-family-no-number  : "Fira Sans", sans-serif;
+@font-family            : "Fira Sans", sans-serif;
+@code-family            : "Fira Mono", Consolas, Menlo, Courier, monospace;
+@heading-color          : @ink-70;
+@text-color             : @ink-80;
+@text-color-secondary   : @ink-70;
+@heading-color-dark     : @ink-90;
+@text-color-dark        : @ink-90;
+@text-color-secondary-dark: @ink-90;
+@font-size-base         : 12px;
+@font-size-lg           : @font-size-base + 2px;
+@line-height-base       : 1.5;
+@border-radius-base     : 2px;
+@border-radius-sm       : 2px;
+
+// The background colors for active and hover states for things like
+// list items or table cells.
+@item-active-bg         : @primary-1;
+@item-hover-bg          : @primary-1;
+
+// ICONFONT
+@iconfont-css-prefix    : anticon;
+@icon-url               : "https://at.alicdn.com/t/font_zck90zmlh7hf47vi";
+
+// LINK
+@link-color             : @primary-color;
+@link-hover-color       : @primary-5;
+@link-active-color      : @primary-7;
+@link-hover-decoration  : none;
+
+// Animation
+@photon-easing          : cubic-bezier(.07,.95,0,1);
+
+// Border color
+@border-color-base      : #d9d9d9;        // base border outline a component
```

Comment:
Is this a non-Design-System colour? And if so, can we make it a Design-System colour?

Reply:
The issue I had was that Ant didn't like using an `rgba` value for this variable, as it is used later in a function, and I guess the function doesn't like `rgba`s.

I've corrected this to now use the appropriate color/opacity, and appropriately adjusted the _other_ variable that relied on this not having opacity!